snuba/query/parser/expressions.py

```diff
@@ -52,6 +56,38 @@ def visit_function_name(self, node: Node, visited_children: Iterable[Any]) -> st
     def visit_column_name(self, node: Node, visited_children: Iterable[Any]) -> Column:
         return Column(None, None, node.text)
 
+    def visit_empty(self, node: Node, visited_children: Iterable[Any]) -> None:
```

Comment:
`-> None` means this does not return anything (like void), which yeah in python kind of means the same but for the type checker it is not the same thing. 
Please return `Optional[Expression]` which allows you to return None.
Also could you please run `mypy --strict snuba/query/parser/expressions.py` I suspect this would not type check even if it behaves properly.

Reply:
Type checking done.